hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -267,6 +278,15 @@ export const PendingTables = () => {
                                 </Button>
                             </div>
                         )}
```

Comment:
Try to use MUI's animation to make the appearance and disappearance of the modal look smooth

Reply:
I think it looks okay lol

Reply:
@ericji1326  Not okay. We should make it perfect and have a nice and consistent user experience. Adding animation isn't that hard. It is also a chance to learn how to add some smooth animation with easy MUI animation API. 

Check it here: 
https://v4.mui.com/zh/components/dialogs/#dialog

Reply:
I am using the popupModal component that I made a couple months ago. I added a fade in animation if that helps.